high_card.rb

```diff
@@ -3,3 +3,16 @@
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
 players = []
+
+
+deck = []
+ranks.each do |a|
+  suits.each do |b|
+     deck.push([a,b])
+  end
+ end
+
+
+# Attempted a deck.shuffle! method with no results
+
+deck.shuffle!
```

Comment:
Nice job getting this far - we are going to go over the State Capitals example together, and it would be awesome if you planned out this project and tried to carry it to completion at some point. First step is plan out everything in as simple words as possible with something like Trello, then go card by card, committing via Git as you go.